### PR TITLE
New OpenShift service mesh section inside integrations

### PR DESCRIPTION
**Pull request by josunect**
I was not sure where to place the new page. 
I've created a new section, "Integrations", but it could be moved directly to a OSSMC section or inside other page. 
Feedback is welcome!

https://deploy-preview-585--kiali.netlify.app/docs/integrations/ossm-console/

### CODE REVIEW

File: content/en/docs/Integrations/ossm-console.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  for OpenShift
+weight: 40
+---
+
+OpenShift Service Mesh Console (OSSMC) is a Kiali integration for OpenShift console based in OpenShift [dynamic plugins](https://docs.openshift.com/container-platform/4.11/web_console/dynamic-plug-ins.html) technology. 
+It integrates part of the Kiali functionality into the OpenShift console, providing visibility of the Service Mesh.
+
+![OSSMC](/images/documentation/integration/ossmc.png)
+ 
+OSSMC was [first released](https://cloud.redhat.com/blog/introducing-the-openshift-service-mesh-console-a-developer-preview) in September 2022, and it is still released as a developer preview, with the purpose of been tested by developers to provide feedback and bug reports.
```
Comment (jshaughn) [inline content/en/docs/Integrations/ossm-console.md]:
```suggestion
OSSMC was [first released](https://cloud.redhat.com/blog/introducing-the-openshift-service-mesh-console-a-developer-preview) in September 2022 as a developer preview.   Please give it a try, provide feedback and let us know of any issues! 
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
Hi @josunect, this looks good.  I like the idea of an Integrations section and I think it makes sense to be at the bottom, underneath the sections that deal with standalone Kiali.  I have one suggested revision for a sentence with a grammar issue.  Feel free to revise my revision :-)

**Comment by josunect** [comment]
> Hi @josunect, this looks good. I like the idea of an Integrations section and I think it makes sense to be at the bottom, underneath the sections that deal with standalone Kiali. I have one suggested revision for a sentence with a grammar issue. Feel free to revise my revision :-)

Thanks, @jshaughn ! It looks more clear and friendly :)

**Review by jshaughn — COMMENTED** [review]
LGTM!  @josunect , the plugin is released and this content is already valid, please also create a backport PR for the `current` branch.

**Review by jshaughn — APPROVED** [review]
Whoops, forgot to approve!

**Pull request merged**